peacebot/core/plugins/Miscellaneous/misc.py

```diff
@@ -1,24 +1,25 @@
+import random
 from datetime import datetime, timedelta
 
 import hikari
 import lightbulb
 
-import peacebot.core.utils.helper_functions as hf
 from peacebot.core.utils.embed_colors import EmbedColors
+from peacebot.core.utils.helper_functions import convert_time, error_handler
 
-from . import convert_time, fetch_scheduler, send_remainder
+from . import fetch_scheduler, send_remainder
 
 misc_plugin = lightbulb.Plugin("Misc", "Miscellaneous Commands for the Bot")
 
 
 @misc_plugin.command
+@lightbulb.option("target", "The target of this command", type=hikari.Member)
```

Comment:
Is there any specific reason for using `target` instead of `member`?

Reply:
`target` would make more sense than member because ultimately we are targetting that member and target can also be used in `Context Menus` command so I put target for easier transition to Context Menu commands.

Reply:
Okay sir, got it.